project/ianlienfa/README.rst

```diff
@@ -0,0 +1,122 @@
+================================================
+BBGym : The branch and bound framework 
+================================================
+
+Project name: BBGym_ 
+.. BBGym_: <https://github.com/ianlienfa/BBGym>
+
+This is a branch and bound problem solving framework for all kinds of OR problems (especially scheduling problems)
+The goal of this framework is to make it easy to toggle between different search strategies, branching strategies and pruning strategies
+The framework aim to construct environment for Reinforcement Learning the branch and search strategies on branch and bound
+
+Problem to Solve
+================
+
+This framework is aim for two goals:
+1. Providing a framework for testing out different search, branch and pruning strategies for some general scheduling problems
+  When researching on dominance relationships or problem-specific heuristics on OR problems, 
+  We often have to first implement the best heuristic and pruning strategy known to us.
+  Then we can test out different search strategies and branch and pruning strategies.
+  This framework is designed to be elastic enough to be able to easily insert and remove strategies
+  
+2. Provide an simple api for RL on branch and bound problem solving (*Main Goal of this Project*)
+
+Prospective Users
+=================
+
+1. Researchers that are interested in the framework and want to try out different search, branch and pruning strategies for OR problems.
+2. Students who would like to solve problem using branch and bound
+3. Industry professionals that would like to try out some ideas before moving onto implementing an state-of-the-art solver
```

Comment:
It will be good to name the specific industry that may use this tool.  Specific description helps further discussions and clarification.  It's OK to speculate.

Reply:
Industry example added, thanks